docs/stratis.txt

```diff
@@ -151,8 +151,9 @@ Mailing list::
 Unknown values::
 	stratis-cli uses the Stratis API. If the API returns values that
 	stratis-cli cannot interpret, stratis-cli will substitute "???". If
-	encountered, upgrading to the latest version of stratis-cli, or filing
-	an issue, is recommended.
+	the API detects that a value is unobtainable, stratis-cli will
```

Comment:
I think that "???" and "FAILURE" mean such different things that the recommendation should be different. For "???" update the CLI and then, if that doens't fix the problem, file an issue, is a good suggestion.

But for "FAILURE", if the value is unobtainable, then something is likely wrong with the pool or blockdev or filesystem. So, the user should be warned somehow, and possibly strongly encouraged to file an issue explaining how they got to the point where list commands had "FAILURE" entries.

So you want to separate out these two different non-values more strongly, at least with separate paragraphs, if not w/ separate headings.

Reply:
Okay, I can split them.  (Though, both seem to fall under the "stratis-cli uses the Stratis API" preamble, but I don't know if it'll be too redundant to state that twice.)

How does this look, just after the original "Unknown Values" heading?
```
Unobtainable values::
        If the API detects that a value is unobtainable, stratis-cli will
        substitute "FAILURE". This may indicate something wrong with the pool,
        blockdev, or filesystem.
```

Reply:
I think that's good. But you made me realize that unknown values preamble is a bit off, too.

Instead of

```
stratis-cli uses the Stratis API. If the API returns values that
	stratis-cli cannot interpret...
```

Try just:

```
If the stratisd D-Bus API returns a value that stratis-cli cannot interpret,...
```

and then for failure
```
If the stratisd D-Bus API returns a message indicating that some value could not be obtained...
```

or something like that.

Reply:
OK; I'm updating it now.